conf/drupal/config/metatag.metatag_defaults.node__campaign_landing.yml

```diff
@@ -11,6 +11,4 @@ tags:
   twitter_cards_image: '[node:field_header:entity:field_image:social_media]'
   twitter_cards_description: '[node:field_meta_description]'
   twitter_cards_type: summary_large_image
-  mass_metatag_organization: '[node:mass_organizations]'
```

Comment:
This is not related to service details and should not be removed.

Reply:
Good catch. Reverted.